packages/crew_profile_base.rb

```diff
@@ -58,33 +48,33 @@ def self.postinstall
 
       # Don't overwrite a custom shell rc
       @_str = "source #{CREW_PREFIX}/etc/profile"
-      FileUtils.touch "#{HOME}/.bashrc" unless File.exist? "#{HOME}/.bashrc"
-      FileUtils.touch "#{HOME}/.zshrc" unless File.exist? "#{HOME}/.zshrc"
-      FileUtils.mv "#{HOME}/.bashrc", "#{HOME}/.bashrc.bak" if `grep -c '#{@_str}' #{HOME}/.bashrc`.to_i.zero?
-      FileUtils.mv "#{HOME}/.zshrc", "#{HOME}/.zshrc.bak" if `grep -c '#{@_str}' #{HOME}/.zshrc`.to_i.zero?
 
       # Write our rc files
       @crew_rcfile = <<~CREW_PROFILE_EOF
-                # DO NOT DELETE THIS LINE
-                # See #{CREW_PREFIX}/etc/profile for further details
-                source #{CREW_PREFIX}/etc/profile
+        # DO NOT DELETE THIS LINE
+        # See #{CREW_PREFIX}/etc/profile for further details
+        source #{CREW_PREFIX}/etc/profile
+
+        # Put your stuff under this comment
 
-                # Put your stuff under this comment
       CREW_PROFILE_EOF
       # Must write directly to HOME and not CREW_DEST_HOME to prevent chromebrew from removing ~/.bashrc during reinstall
-      IO.write("#{HOME}/.bashrc", @crew_rcfile) unless File.exist? "#{HOME}/.bashrc"
-      IO.write("#{HOME}/.zshrc", @crew_rcfile) unless File.exist? "#{HOME}/.zshrc"
-
-      # Don't overwrite a custom shell rc part 2
-      system "cat #{HOME}/.bashrc.bak >> #{HOME}/.bashrc" if File.exist? "#{HOME}/.bashrc.bak"
-      system "cat #{HOME}/.zshrc.bak >> #{HOME}/.zshrc" if File.exist? "#{HOME}/.zshrc.bak"
-
-      puts
-      puts 'IT IS IMPERATIVE TO THE FUNCTION OF CHROMEBREW FOR'.lightblue
-      puts "#{HOME}/.bashrc to contain the line 'source #{CREW_PREFIX}/etc/profile'.".lightblue
-      puts 'This should have happened automatically.'.lightblue
-      puts "If the line doesn't exist, add it now and report a bug at https://github.com/chromebrew/crew-profile-base/issues.".lightblue
-      puts
+      if File.exist? "#{HOME}/.bashrc"
+        if `grep -c "#{@_str}" #{HOME}/.bashrc`.to_i.zero?
+          old_bashrc = IO.read "#{HOME}/.bashrc"
+          IO.write "#{HOME}/.bashrc", @crew_rcfile + old_bashrc
+        end
+      else
+        IO.write("#{HOME}/.bashrc", @crew_rcfile)
+      end
+      if File.exist? "#{HOME}/.zshrc"
+        if `grep -c "#{@_str}" #{HOME}/.zshrc`.to_i.zero?
+          old_zshrc = IO.read "#{HOME}/.zshrc"
+          IO.write "#{HOME}/.zshrc", @crew_rcfile + old_zshrc
```

Comment:
Should we have a puts line to let the user know their shell config has been backed up?